src/XenobladeX/Mods/BattleEnemyStats/rules.txt

```diff
@@ -2,8 +2,11 @@
 titleIds = 0005000010116100,00050000101C4C00,00050000101C4D00
 name = "Enemy Statistics modificator"
 path = "Xenoblade Chronicles X/Mods/Battle/Enemy Statistics modificator"
-description = Apply a percent modificator on enemies stats (HP, Melee Attack, Melee Accuracy, Ranged Attack, Ranged Accuracy, Potential and Evasion). You can consult the new values in the Bestiary menu.
-version = 4
+description = Apply a percent modificator on enemies stats (HP, Melee Attack, Melee Accuracy, Ranged Attack, Ranged Accuracy, Potential and Evasion).|You can consult the new values in the Bestiary menu.
+version = 5
+
+[Default]
+$mod = 0.001
```

Comment:
probably should add a preset like "Keep at default" so that this 0.001 gets used